src/getCoinCombination.test.js

```diff
@@ -7,4 +7,28 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should be return '[1, 1, 0, 0]' when argument '6'`, () => {
```

Comment:
The test description should be corrected to 'should return [1, 1, 0, 0] when argument is 6' for clarity and grammatical correctness.